part-spec/common/thermals.json

```diff
@@ -0,0 +1,81 @@
+{
+    "$id": "https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/common/thermals.json",
```

Comment:
One thing that might be worthwhile is getting this reviewed by a power/thermal engineer - I've worked with a few folks at Intel over the years, not sure if they would be willing to give a quick read-over?

Reply:
Good idea.